### PR TITLE
Add motif env vars

### PR DESCRIPTION
**Pull request by p6l-richard**
## What does this PR do?

Adds motif.land (where we host our /docs) environment variables to the turbo.json

Note: The actual usage is in the website repository (see https://github.com/calcom/website/pull/451).

**Environment**: Staging(main branch) / Production

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [ ] Visiting the preview site on https://github.com/calcom/website/pull/451 for <url>/sitemap.xml should _not_ return a 500

### CODE REVIEW

File: packages/embeds/embed-core/src/embed-iframe.ts [hunk 1]
```diff
@@ -344,13 +344,15 @@ function keepParentInformedAboutDimensionChanges() {
     // Use, .height as that gives more accurate value in floating point. Also, do a ceil on the total sum so that whatever happens there is enough iframe size to avoid scroll.
     const contentHeight = Math.ceil(
       parseFloat(mainElementStyles.height) +
-      parseFloat(mainElementStyles.marginTop) +
-      parseFloat(mainElementStyles.marginBottom));
+        parseFloat(mainElementStyles.marginTop) +
+        parseFloat(mainElementStyles.marginBottom)
+    );
```
Comment (p6l-richard) [inline packages/embeds/embed-core/src/embed-iframe.ts:349]: auto-format (not sure where this comes from 🤔 )

File: packages/embeds/embed-core/src/embed-iframe.ts [hunk 2]
```diff
@@ -344,13 +344,15 @@ function keepParentInformedAboutDimensionChanges() {
...
     const contentWidth = Math.ceil(
       parseFloat(mainElementStyles.width) +
-      parseFloat(mainElementStyles.marginLeft) +
-      parseFloat(mainElementStyles.marginRight));
-    
+        parseFloat(mainElementStyles.marginLeft) +
+        parseFloat(mainElementStyles.marginRight)
+    );
+
```
Comment (p6l-richard) [inline packages/embeds/embed-core/src/embed-iframe.ts:355]: auto-format (not sure where this comes from 🤔 )

File: packages/prisma/seed-app-store.config.json [hunk 3]
```diff
@@ -203,7 +203,8 @@
     "categories": ["video"],
     "slug": "facetime",
     "type": "facetime_video",
-    "isTemplate": false},
+    "isTemplate": false
+  },
```
[the remark this answers is not in the document]
Comment (zomars) [reply packages/prisma/seed-app-store.config.json:207]: Let's undo with `git checkout origin/main packages/prisma/seed-app-store.config.json`

File: packages/embeds/embed-core/src/embed-iframe.ts (hunk 2 above)
Comment (zomars) [reply packages/embeds/embed-core/src/embed-iframe.ts:355]: Let's undo with `git checkout origin/main packages/embeds/embed-core/src/embed-iframe.ts`

**Review by zomars — DISMISSED** [review]

**Review by emrysal — APPROVED** [review]
This is good to go in 🙏

**Pull request merged**